test/functionalTest/cases/0787_cprs_full_functional/update_false_registration_and_one_more_value_in_ngsiv2.test

```diff
@@ -179,11 +179,16 @@ Date: REGEX(.*)
 
 03. Update/UPDATE E1/T1/A1+A2 in CB
 ===================================
-HTTP/1.1 204 No Content
-Content-Length: 0
+HTTP/1.1 404 Not Found
+Content-Length: 92
+Content-Type: application/json
 Fiware-Correlator: REGEX([0-9a-f\-]{36})
 Date: REGEX(.*)
 
+{
+    "description": "Attributes that were not updated: [ E1-T1 : [A1] ]",
```

Comment:
This change in .test (and some other similar) correspond to the following entry in CNR: "Fix: using 404 Not Found - PartialUpdate error when  NGSIv2 forwarded update is partially done (previously 204 No Content was used, without providing actual informaton about the partial update)".